### ISSUE TITLE
App can't display 64MP images

### ISSUE THREAD

**Issue by sniederb** [sniederb/photocompare#2]
When attempting to display/compare 64MP images, the image simply doesn't render.

The trace

```
2020-01-23 09:57:57.795 15064-15064/? E/AndroidRuntime: FATAL EXCEPTION: main
    Process: ch.want.imagecompare, PID: 15064
    java.lang.RuntimeException: Canvas: trying to draw too large(128288256bytes) bitmap.
        at android.view.DisplayListCanvas.throwIfCannotDraw(DisplayListCanvas.java:229)
        at android.view.RecordingCanvas.drawBitmap(RecordingCanvas.java:98)
        at com.bumptech.glide.load.resource.bitmap.GlideBitmapDrawable.draw(GlideBitmapDrawable.java:101)
        at android.widget.ImageView.onDraw(ImageView.java:1360)
        at android.view.View.draw(View.java:20207)
        at android.view.View.updateDisplayListIfDirty(View.java:19082)
        at android.view.View.draw(View.java:19935)
        at android.view.ViewGroup.drawChild(ViewGroup.java:4333)
        at android.view.ViewGroup.dispatchDraw(ViewGroup.java:4112)
        at android.view.View.draw(View.java:20210)
        at androidx.viewpager.widget.ViewPager.draw(ViewPager.java:2426)
        at android.view.View.updateDisplayListIfDirty(View.java:19082)
        at android.view.View.draw(View.java:19935)
        at android.view.ViewGroup.drawChild(ViewGroup.java:4333)
        at android.view.ViewGroup.dispatchDraw(ViewGroup.java:4112)
        at android.view.View.updateDisplayListIfDirty(View.java:19073)
```

indicates a case of https://github.com/bumptech/glide/issues/3775

**Comment by clycheng2015** [comment]
+1

**Comment by sniederb** [comment]
Local tests look good, switching to a different view library did the trick (released with 1.0.18)
